src/App.css

```diff
@@ -0,0 +1,39 @@
+.wrapper {
+  width: 375px;
+  background-color: rgb(241, 89, 114);
+  margin: auto;
+}
```

Comment:
### Design on Iphone XR
![design-on-iphone-XR (copy)](https://user-images.githubusercontent.com/93039359/187934909-2168faf5-5b32-4336-a9fb-30ec0d06f1f7.png)

### Design on Iphone 12 Pro
![design-on-iphone-12Pro (copy)](https://user-images.githubusercontent.com/93039359/187934925-9ec930ce-cae8-43cc-a85d-425e16a413fe.png)

- [ ] I know that this project is not responsive but it should be responsive enough to fit on all the mobile and table screens  so please make it responsive :blush: You can see in the above images that purple boxes with white spacing show that your design is not fitting on other phone models which is not making it professional :thinking:

### You can get an idea from this website :+1:
https://stock-metrics-clint.netlify.app/